transcendence-app/test/auth/auth.e2e-spec.ts

```diff
@@ -0,0 +1,74 @@
+import { Test, TestingModule } from '@nestjs/testing';
+import { AuthController } from '../../src/auth/auth.controller';
+import { HttpStatus, INestApplication } from '@nestjs/common';
+import { AuthModule } from '../../src/auth/auth.module';
+import * as request from 'supertest';
+import { ConfigModule, ConfigService } from '@nestjs/config';
+import {
+  EnvironmentVariables,
+  validate,
+} from '../../src/config/env.validation';
+import * as session from 'express-session';
+import * as ConnectMemcached from 'connect-memcached';
+import * as passport from 'passport';
+
+describe('[Feature] Auth - /auth', () => {
```

Comment:
These are the tests from `auth.controller.spec.ts`. I've moved them here because they are e2e tests that require PostgreSQL and Memcached.